pylint_odoo/checkers/itp_checks.py

```diff
@@ -138,6 +143,29 @@ def _check_manifest_image(self):
             else:
                 return False
 
+    @utils.check_messages('readme-versions')
+    def _check_readme_versions(self):
+        readme_file = os.path.isfile(os.path.join(self.module_path, 'README.rst'))
+        if readme_file:
+            for line in self.readme_text:
+                urls = re.findall(
+                    'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\),]|(?:%[0-9a-fA-F][0-9a-fA-F]))+', line)
+                if urls != []:
+                    for url in urls:
+                        odoo_version = re.search('/(\d+.\d)/', url)
+                        if odoo_version:
+                            valid_odoo_version = self.linter._all_options[
+                                'valid_odoo_versions'].config.valid_odoo_versions[0]
```

Comment:
You can compute variable valid_odoo_version once. No need to do it in every iteration

Reply:
will to do!